web/components/recipes.templ

```diff
@@ -174,6 +174,34 @@ templ addRecipe() {
 					standard, the <a href="https://www.mastercook.com/" target="_blank" class="link">MasterCook</a> file
 					format, or as plain text files.
 				</p>
+				<p>
+					You can also download recipe schema files directly using the
+					<a
+						class="link tooltip"
+						data-tip="Simply drag this link to your bookmarks bar, and click the bookmark while on a recipe website.  If a recipe schema downloads successfully, you can import it here."
+						href="javascript:(function(){
+							const recipeCount = 0;
+							jsonSchemaScripts = ([...document.querySelectorAll('script[type=%22application/ld+json%22]')]);
```

Comment:
This line is missing its variable declarator. You can change it to
`const jsonSchemaScripts = ([...document.querySelectorAll('script[type=%22application/ld+json%22]')]);`